docs/dev/build-instructions/build_static.md

```diff
@@ -129,6 +129,22 @@ Note also that you **have** to use the command prompt and **not** the Developer
 
 If you are on a 64bit system, then you'll know that you have opened the correct prompt, if it prints `Environment initialized for: 'x64'`.
 
+However, you can user powershell 64bit if you run the following script prior to cmake. You can either run it as required, or include it in your powershell profile. 
+
+```powershell
+# Shift context to VS
+Push-Location "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\"
```

Comment:
Maybe the text above should mention that this path has to be adapted to your specific VS version?There doesn't happen to be a environment variable that we can query for this location, is there?

Reply:
Agreed - I considered writing the registry script for this, but even my own dev environment with 2019 references the 2015 location. I'm not sure it will be reliable, so hoping the comment and brief instruction suffices.